Connector/server.py

```diff
@@ -142,7 +137,15 @@ async def onShutdown(app):
 def runServer():
 
     app = WebApp()
+    cors = aiohttp_cors.setup(app, defaults={
+        "*": aiohttp_cors.ResourceOptions(
+            expose_headers="*",
+            allow_headers="*",
+        )
+    })
     app.add_routes([web.post('/rpc', rpcServerHandler)])
+    for route in list(app.router.routes()):
```

Comment:
Why is cors configured before adding the ws endpoint?

Reply:
Because browsers don't implement CORS protection for WS so it's not necessary